plugins/inputs/all/all.go

```diff
@@ -19,7 +19,7 @@ import (
 	_ "github.com/influxdata/telegraf/plugins/inputs/disque"
 	_ "github.com/influxdata/telegraf/plugins/inputs/dmcache"
 	_ "github.com/influxdata/telegraf/plugins/inputs/dns_query"
-	_ "github.com/influxdata/telegraf/plugins/inputs/docker"
+	//_"github.com/influxdata/telegraf/plugins/inputs/docker"
```

Comment:
Why was commenting this out necessary?

Reply:
there were some errors due to some dependency. So i commented it out for now.

Reply:
Before this is merged, it needs to be fixed.